iib/web/migrations/versions/4c9db41195ec_add_merge_index_image_api_endpoint.py

```diff
@@ -0,0 +1,67 @@
+"""Add merge-index-image api endpoint.
+
+Revision ID: 4c9db41195ec
+Revises: bc29053265ba
+Create Date: 2020-09-28 23:06:43.267716
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '4c9db41195ec'
+down_revision = 'bc29053265ba'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'request_merge_index_image',
+        sa.Column('id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.Column('binary_image_id', sa.Integer(), nullable=False),
+        sa.Column('binary_image_resolved_id', sa.Integer(), nullable=True),
+        sa.Column('index_image_id', sa.Integer(), nullable=True),
+        sa.Column('source_from_index_id', sa.Integer(), nullable=False),
+        sa.Column('source_from_index_resolved_id', sa.Integer(), nullable=True),
+        sa.Column('target_index_id', sa.Integer(), nullable=True),
+        sa.Column('target_index_resolved_id', sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(['binary_image_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['binary_image_resolved_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['id'], ['request.id'],),
+        sa.ForeignKeyConstraint(['index_image_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['source_from_index_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['source_from_index_resolved_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['target_index_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['target_index_resolved_id'], ['image.id'],),
+        sa.PrimaryKeyConstraint('id'),
+    )
+
+    op.create_table(
+        'bundle_deprecation',
+        sa.Column('merge_index_image_id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.Column('bundle_id', sa.Integer(), autoincrement=False, nullable=False),
+        sa.ForeignKeyConstraint(['bundle_id'], ['image.id'],),
+        sa.ForeignKeyConstraint(['merge_index_image_id'], ['request_merge_index_image.id'],),
+        sa.PrimaryKeyConstraint('merge_index_image_id', 'bundle_id'),
+        sa.UniqueConstraint('merge_index_image_id', 'bundle_id'),
```

Comment:
Is there a way to name the unique constraint in case it needs to be altered in the future?